setup.py

```diff
@@ -90,7 +90,6 @@ def get_version():
             "pyomo/core/expr/numvalue.pyx",
             "pyomo/core/expr/numeric_expr.pyx",
             "pyomo/core/expr/logical_expr.pyx",
-            "pyomo/core/expr/visitor_expr.pyx",
```

Comment:
This should not be removed, but instead be changed to `"pyomo/core/expr/visitor.pyx"`

Reply:
Please see my second comment.

Reply:
Doh.  You are, of course, correct.  We should probably leave it in as a comment, though: the visitor file has all the routines for walking expression trees and a good chunk of the benefit of cython will come from cythonizing that file, so we should try and get the circular dependence resolved soon.